### PR TITLE
feature(business_central): Remove cols marked for deprecation in AL code  Q3 2024

### PR DESCRIPTION
**Pull request by fivetran-inagurimunisaihaneesh**
Links https://fivetran.height.app/T-757136

As part of Business central service checkup for Q3 2024, cleaning up the fields from standard tables that will be deprecated in future releases by Business Central.

This is not a breaking change for customers and no customer comms are required for this change.

### CODE REVIEW

File: InitialSetup.al [hunk 1]
```diff
@@ -432,14 +432,13 @@ page 90003 "G/L Account"
...
-    EntityName = 'generalLedgerAccount'; // page Name - schema - destination
-    EntitySetName = 'generalLedgerAccounts'; // url 
+    EntityName = 'generalLedgerAccount';
+    EntitySetName = 'generalLedgerAccounts';
     InsertAllowed = false;
     ModifyAllowed = false;
     ODataKeyFields = SystemId;
     PageType = API;
-    SourceTable = "G/L Account"; // no - access
-    DataAccessIntent = ReadOnly;
```
Comment (fivetran-mandeepsingh) [inline InitialSetup.al:442]: Why is this removed?

Comment (fivetran-inagurimunisaihaneesh) [reply InitialSetup.al:442]: Unintended change; reverted back

**Review by fivetran-mandeepsingh — APPROVED** [review]
LGTM
